bluesky/tests/test_suspenders.py

```diff
@@ -5,7 +5,7 @@
                                 SuspendBoolLow,
                                 SuspendFloor,
                                 SuspendCeil,
-                                SuspendInBand,
+                                SuspendWhenOutsideBand,
                                 SuspendOutBand)
```

Comment:
should this also be renamed?

Reply:
It's a deprecated class, it will eventually be removed.

Reply:
Do you think we should un-document it at this time as well?

Reply:
Oh, nevermind. You already did!